src/components/elements/dropdown-search.js

```diff
@@ -28,7 +28,7 @@ const DropdownInput = styled.input`
     }
 
     @media ${device.tabletL} {
-        font-size: 1.75rem;
+        font-size: ${(props) => (props.is_affiliate ? '1.6rem' : '1.75rem')};
```

Comment:
Please accept a more general prop like `tabletL_size` or something like that.

Reply:
Removed `is_affiliate` prop with relevant style props.